Sources/SmithyChecksumsAPI/ChecksumAlgorithm.swift

```diff
@@ -7,4 +7,14 @@
 
 public enum ChecksumAlgorithm {
     case crc32, crc32c, sha1, sha256, md5
+
+    public func toString() -> String {
+        switch self {
+        case .crc32: return "crc32"
+        case .crc32c: return "crc32c"
+        case .sha1: return "sha1"
+        case .sha256: return "sha256"
+        case .md5: return "md5"
```

Comment:
should we add an `.unknown(String)` case to this enum that just returns the unknown string when queried?